### PR TITLE
add some info for separate compose and execute commands

### PR DESCRIPTION
**Pull request by i-norden**

### CODE REVIEW

File: documentation/composeAndExecute.md [hunk 1]
```diff
@@ -150,3 +138,40 @@ func (e exporter) Export() []interface1.EventTransformerInitializer, []interface
...
+[storage](../../staging/libraries/shared/watcher/storage_watcher.go#L53),
+or [contract](../../staging/libraries/shared/watcher/contract_watcher.go#L68) watcher execution modes
+3. Create db migrations to run against vulcanizeDB so that we can store the transformer output
+    * Do not `goose fix` the transformer migrations, this is to ensure they are always ran after the core vulcanizedb migrations which are kept in their fixed form
+    * Specify migration locations for each transformer in the config with the `exporter.transformer.migrations` fields
+    * If the base vDB migrations occupy this path as well, they need to be in their `goose fix`ed form
+    as they are [here](../../staging/db/migrations)
+
+To update a plugin repository with changes to the core vulcanizedb repository, replace the vulcanizedb vendored in the plugin repo (`plugin_repo/vendor/github.com/vulcanize/vulcanizedb`)
+with the newly updated version
```
Comment (rmulhol) [inline documentation/composeAndExecute.md]: Is it possible to do this with `dep` now that the repo is public?

Comment (i-norden) [reply documentation/composeAndExecute.md]: Yeah it is, thank you!

**Comment by i-norden** [comment]
Made that change and also moved [this](https://github.com/vulcanize/vulcanizedb/blob/dbd1d1baf9f16db4aa0d7aa2986e95b738f30b9b/documentation/transformers.md) to a new file linked to from the bottom of this section in the main readme [here](https://github.com/vulcanize/vulcanizedb/tree/docs#data-transformation)

**Comment by i-norden** [comment]
Also added a note about that flaky test and took a closer look at what might be causing it but haven't figured it out yet.

**Pull request merged**